kuzzleio/kuzzle
# Add defaut input.headers (empty object) to websocket request

## What does this PR do ?

This PR add default headers to Request input for websocket protocols (websocket and socketio).  
The default headers are an empty object.  

Fix https://github.com/kuzzleio/kuzzle/issues/1067

### How should this be manually tested?

  - Step 1 : Add a plugin to Kuzzle : `git clone https://github.com/kuzzleio/kuzzle-core-plugin-boilerplate plugins/enabled/core-plugin`
  - Step 2 : In `plugins/enabled/core-plugin/lib/index.js#L248`, change `request.input.body.createdAt = Date.now();` by `console.log('LOOK HERE', request.input.headers);`
  - Step 3 : Run docker-compose: `docker-compose -f docker-compose/dev.yml up`
  - Step 3 :  Create a document in the admin console, you should see `LOOK HERE {}` in your terminal
